<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>矩阵计算器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>创建矩阵</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="rbMatrixA">
             <property name="text">
              <string>矩阵 A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbMatrixB">
             <property name="text">
              <string>矩阵 B</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>行数:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnRows">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>列数:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnCols">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnCreateMatrix">
           <property name="text">
            <string>创建矩阵</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="btnLoadMatrix">
             <property name="text">
              <string>加载矩阵</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSaveMatrix">
             <property name="text">
              <string>保存矩阵</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="cmbOperation"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnCalculate">
           <property name="text">
            <string>计算</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>矩阵</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lblMatrixA">
           <property name="text">
            <string>矩阵 A:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txtMatrixA">
           <property name="plainText">
            <string/>
           </property>
           <property name="placeholderText">
            <string>按空格分隔元素，按回车分隔行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMatrixB">
           <property name="text">
            <string>矩阵 B:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txtMatrixB">
           <property name="plainText">
            <string/>
           </property>
           <property name="placeholderText">
            <string>按空格分隔元素，按回车分隔行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblResult">
           <property name="text">
            <string>结果:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txtResult">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
